neo/Consensus/ConsensusService.cs

```diff
@@ -272,6 +269,38 @@ private void OnPersistCompleted(Block block)
             InitializeConsensus(0);
         }
 
+        private void ObtainTransactionsForConsensus()
```

Comment:
Why do you want to separate this function?

Reply:
I can remove separating it. Originally, I had a reason, but it was removed in updates. I can remove splitting it now.

Reply:
Removed it from being a separate function.